server/command.go

```diff
@@ -262,12 +284,111 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 
 func (p *Plugin) runSettingCommand(args *model.CommandArgs, params []string, user *model.User) (string, error) {
 	if len(params) == 0 {
+		restrict, _, err := p.checkChannelPreference(args.ChannelId)
+		if err != nil {
+			p.API.LogError("Unable to check channel preference", "ChannelID", args.ChannelId, "Error", err.Error())
+			return "Error occurred while viewing zoom settings", nil
+		}
+
+		if restrict {
+			return "Updating zoom settings is disabled for this current channel.", nil
+		}
+
 		if err := p.sendUserSettingForm(user.Id, args.ChannelId, args.RootId); err != nil {
 			return "", err
 		}
 		return "", nil
 	}
-	return fmt.Sprintf("Unknown Action %v", ""), nil
+	return actionUnkown, nil
+}
+
+func (p *Plugin) runChannelSettingsCommand(args *model.CommandArgs, params []string, user *model.User) (string, error) {
+	if !p.API.HasPermissionTo(args.UserId, model.PermissionManageSystem) {
+		return "Unable to execute the command, only system admins have access to execute this command.", nil
+	}
```

Comment:
I'm thinking users that have permission to manage the channel can run `channel-settings`, but sysadmin privileges should be needed for `channel-settings list`